sycl/test/basic_tests/parallel_for_range_roundup.cpp

```diff
@@ -0,0 +1,197 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple  %s -o %t.out
+// RUN: env SYCL_PARALLEL_FOR_RANGE_ROUNDING_TRACE=1 %GPU_RUN_PLACEHOLDER %t.out %GPU_CHECK_PLACEHOLDER
+
+#include <CL/sycl.hpp>
+
+using namespace sycl;
+
+range<1> Range1 = {0};
+range<2> Range2 = {0, 0};
+range<3> Range3 = {0, 0, 0};
+
+void check(const char *msg, size_t v, size_t ref) {
+  std::cout << msg << v << std::endl;
+  assert(v == ref);
+}
+
+int try_item1(size_t size) {
```

Comment:
Why do all of these functions have return type of `int`? I don't see the return value to be used.

Reply:
Regarding previously compiled apps with a newer library that accompanies the headers changed by this PR. Older apps will execute as they did before. This PR makes changes to the SYCL translation through header file changes only. There is no dependence on the library.